openshift_sti_images/python.adoc

```diff
@@ -0,0 +1,191 @@
+= Python
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+The OpenShift https://github.com/openshift/sti-python/tree/master/3.3[Python] image repository contains the sources and Dockerfiles for building various versions of Python platform images. The resulting images can be run either by Docker or using https://github.com/openshift/source-to-image[STI].
+
+== Versions
+Currently, OpenShift only supports version https://github.com/openshift/sti-python/tree/master/3.3[3.3] of Python.
```

Comment:
it's not an openshift support statement...we support basically anything.  would not want someone to read this and think they can't run a python 2.7 image on openshift.